build/tools/README.md

```diff
@@ -6,4 +6,4 @@ description: A builder without the right tool is just a fool
 
 <mark style="color:red;">Section to be reviewed and updated</mark>
 
-<table data-card-size="large" data-view="cards"><thead><tr><th></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td><strong>Persistence JS</strong></td><td><a href="persistence-js.md">persistence-js.md</a></td></tr><tr><td><strong>Persistence SDK</strong></td><td><a href="persistence-sdk.md">persistence-sdk.md</a></td></tr><tr><td><strong>Persistence CLI</strong></td><td><a href="persistence-cli.md">persistence-cli.md</a></td></tr></tbody></table>
+<table data-card-size="large" data-view="cards"><thead><tr><th></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td><strong>Persistence JS</strong></td><td><a href="persistence-js.md">persistence-js.md</a></td></tr><tr><td><strong>Persistence SDK</strong></td><td><a href="persistence-sdk.md">persistence-sdk.md</a></td></tr><tr><td><strong>Persistence CLI</strong></td><td><a href="persistence-cli.md">persistence-cli.md</a></td></tr><tr><td><strong>Cosmovisor Tool</strong></td><td><a href="cosmovisor.md">cosmovisor.md</a></td></tr></tbody></table>
```

Comment:
I have added this doc snippet in the "Join a mainnet" and "join a Testnet" page itself rather than a separate page just for a small feature and these pages already had details related to cosmovisor. In latest commit, I have made minor point-wise bullets within the existing docs and appropriately added the feature details. Please check and let me know your review.